docs/public/_redirects

```diff
@@ -20,10 +20,9 @@
 /r/issue-template-latest https://codesandbox.io/s/material-ui-issue-latest-s2dsx
 /r/ts-issue-template https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAKjgQwM5wEoFNkGN4BmUEIcA5FDvmQNwBQokscA3nXHAPSdwwAWWOLhKQAdllEx0ATwgBXOHNRYAJnQC+cIiXIABEHOCcQyGFijBkAGzJ06BOaPzAIouABEsICAAoAlKzsXDwAmvJQQhAqWBpAA 302
 /r/custom-component-variants /customization/components/#adding-new-component-variants
-/r/x-license https://mui.com/store/items/material-ui-pro/
 /r/migration-v4 /guides/migration-v4/
 
-/x/license/ https://docs.google.com/document/d/1nkd5JP5wCefo6UwpVi6PnbTaw8DxhstkCZlfHV5ZgfI/edit?usp=sharing 301
+/x/license https://mui.com/store/legal/mui-x-eula/ 301
```

Comment:
What's the use case for a redirection? Why not link to https://mui.com/store/legal/mui-x-eula/ directly? 

```suggestion
```

Reply:
Agree, I think that having /x/license under the legacy section of `_redirects` file (chronologically ordered) and keeping `/x/license/` instead of `/x/license` would be perfect